### DISCUSSION TITLE
Clock ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#430] [Q&A] [answered]
Hi !

it is possible to customize clock like this 👍 

I have licence for Atomic Clock Alarm but if I put Explorer_Patcher adapt for Win 10 (in Win 11), the problem is the Windhawk doesn't work... 

I have Win 11 on my computer.

Thanks to tell me !
![atomic-alarm-clock](https://github.com/ramensoftware/windhawk-mods/assets/27967268/4ea746b5-9d5a-44ea-8cda-ee30cb85bb5d)

**Comment by m417z** [comment, marked as answer]
You can look at the options of [Taskbar Clock Customization](https://windhawk.net/mods/taskbar-clock-customization) to see which customizations are possible. Parameters such as color, size, and font can only be customized for the new Windows 11 taskbar. Otherwise, only the text can be customized.

In other words, no, Windhawk doesn't have a mod that allows to have a fancy clock like on your image.

**Comment by sypqys** [comment]
I don't know how I have this ? 
![3JiXwjT](https://github.com/ramensoftware/windhawk-mods/assets/27967268/25c5e617-eb90-4b8f-b4f6-5cfa28e9c008)

**Reply by m417z** [reply]
I changed parameters such as the color, the size, the font width and the font type.
Note that as said in the settings, and as I mentioned in a previous comment, it won't work with ExplorerPatcher, only with the new Windows 11 taskbar.

**Reply by sypqys** [reply]
I have uninstall Explorer_Patcher, I don't run anymore on my computer.
Thanks
